common/models/templates/build.sh

```diff
@@ -29,43 +29,31 @@ builder_describe "Builds the predictive-text model template implementation modul
 
 builder_describe_outputs \
   configure          /node_modules \
-  build              build/lib/index.mjs
+  build              build/obj/index.js
 
 builder_parse "$@"
 
-### CONFIGURE ACTIONS
-
-if builder_start_action configure; then
-  verify_npm_setup
-  builder_finish_action success configure
-fi
-
-### CLEAN ACTIONS
-
-if builder_start_action clean; then
-  rm -rf build/
-  builder_finish_action success clean
-fi
+builder_run_action configure verify_npm_setup
+builder_run_action clean rm -rf build/
```

Comment:
```suggestion
builder_run_action configure verify_npm_setup
builder_run_action clean     rm -rf build/
```

Table formatting makes it easier to spot the actual actions :grin: